cmd/kots/cli/install.go

```diff
@@ -163,6 +163,11 @@ func InstallCmd() *cobra.Command {
 
 			upstream := pull.RewriteUpstream(args[0])
 
+			preferredChannelSlug := ""
+			if len(args) > 1 {
+				preferredChannelSlug = args[1]
```

Comment:
This should be in the same arg as app slug, as shown in the docs: https://docs.replicated.com/reference/kots-cli-install#examples

```
kubectl kots install kots-sentry/stable
```

Reply:
+1

Reply:
yes, and it's optional. should default to "stable"

Reply:
ah cool 👍 will send along one more update to use the default.